### PR TITLE
Add code conduct page

### PR DESCRIPTION
**Pull request by att55**
## Issue
- close https://github.com/DroidKaigi/conference-app-2022/issues/189

## Overview (Required)

Add "code conduct" under "license".

### CODE REVIEW

**Comment by att55** [comment]
@takahirom OK. I will add WIP to title of this PR. 🙆🏼‍♂️

**Comment by takahirom** [comment]
Can you check the conflict?🙏

File: feature-about/src/main/res/values-en/strings.xml [hunk 1]
```diff
@@ -7,4 +7,5 @@
     <string name="about_staff">Staff List</string>
     <string name="about_privacy">Privacy Policy</string>
     <string name="about_license">License</string>
+    <string name="about_code_conduct">Code Conduct</string>
```
Comment (kittinunf) [inline feature-about/src/main/res/values-en/strings.xml:10]: This should be moved to `core.model/commonMain/resources`

Comment (att55) [reply feature-about/src/main/res/values-en/strings.xml]: Automatically moved by merging main branch 👍🏼

**Comment by att55** [comment]
@takahirom 
Fixed conflicts! Please let me confirm two things. 🙏🏼
1. Is the position of "code of conduct" bottom (under "license") in about screen?
2. How's about "privacy policy"?
https://github.com/DroidKaigi/conference-app-2022/issues/189#issue-1361411972

**Comment by takahirom** [comment]
How about this order.

Code of conduct
Privacy policy

From web
image

**Comment by att55** [comment]
@takahirom Is this order correct?
```
Access
Staff List
License
Code of conduct
Privacy Policy
App Version
```

**Comment by att55** [comment]
And how's about "privacy policy"? 
Should I develop in this PR or another PR (issue)?

**Comment by takahirom** [comment]
There is the issue 👀 
https://github.com/DroidKaigi/conference-app-2022/issues/188

**Comment by takahirom** [comment]
LGTM! Thanks for your cooperation.

**Pull request merged**